Containerfile

```diff
@@ -2,18 +2,21 @@
 FROM registry.fedoraproject.org/fedora:latest AS fedora-build
 
 RUN dnf -yq install \
-              createrepo_c \
               fedpkg \
               gpg \
               jq \
               make \
               ostree \
               patch \
+              python3-pip \
               python3-jinja2-cli \
               rpm-build \
               rpm-sign \
               rsync
 
+# the default createrepo_c provided by Fedora has legacy hashes disabled, the one
+# on PyPI does not (because we need it)
```

Comment:
What happens if that package would be upgraded also? Want to add a version constraint?

Reply:
Nah.  It's actually good for something like that to make a bit of noise early on.

But also, the Python package is basically our responsibility in the first place, so it's unlikely to change out from under us.